external-crates/move/crates/move-analyzer/editors/code/CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ To be added to the group of maintainers allowed to release new versions of the m
 2. Sign in with your Microsoft account's email address. If using a corporate email address, you'll go through an authentication flow specific to your organization (two-factor authentication, for example).
 3. Once you've signed in, contact someone who is already in the existing group of maintainers, and ask them to [please add you to this list of members](https://marketplace.visualstudio.com/manage/publishers/move). If you do not know anyone in this group personally, run `git log -- language/move-analyzer/editors/code` and send an email to one or more of the people who have committed to this directory.
 
```

Comment:
Changes in this file are unrelated, but while I was creating a token to publish this new version, I realized that the instructions are not quite accurate and decided to fix them right away